samples/bluetooth/radio_coex_1wire/src/main.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <sys/printk.h>
+#include <sys/__assert.h>
+
+#include <bluetooth/bluetooth.h>
+#include <console/console.h>
+
+#include <hal/nrf_radio.h>
+#include <hal/nrf_timer.h>
+#include <helpers/nrfx_gppi.h>
+#include <nrfx_gpiote.h>
+#include <nrfx_timer.h>
+#include <nrfx_ppi.h>
+#include <hal/nrf_gpio.h>
+
+#define STACKSIZE CONFIG_MAIN_STACK_SIZE
+#define THREAD_PRIORITY K_LOWEST_APPLICATION_THREAD_PRIO
+#define M_COUNTER NRF_TIMER1
+
+#define ZEPHYR_USER_NODE DT_PATH(zephyr_user)
+#if (DT_NODE_HAS_PROP(ZEPHYR_USER_NODE, coex_pta_grant_gpios) && \
+	DT_NODE_HAS_STATUS(DT_NODELABEL(radio_coex_1w), okay))
+
+/* PTA input */
+#define APP_GRANT_GPIO_PIN NRF_DT_GPIOS_TO_PSEL(ZEPHYR_USER_NODE, coex_pta_grant_gpios)
+#else
+#error "Unsupported board: see README and check /zephyr,user node"
+#define APP_GRANT_GPIO_PIN 0
+#endif
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+};
+
+static void print_welcome_message(void)
+{
+	if (console_init()) {
+		__ASSERT(false, "Failed to initialise console");
+	}
+
+	printk("-----------------------------------------------------\n");
+	printk("This sample illustrates the 1Wire coex interface\n");
+	printk("The number of radio request every second is printed\n");
```

Comment:
The number of radio events is printed every second. (or.. sent radio packets ..)